pandas/core/dtypes/dtypes.py

```diff
@@ -1155,7 +1155,7 @@ class IntervalDtype(PandasExtensionDtype):
 
     _cache_dtypes: dict[str_type, PandasExtensionDtype] = {}
 
-    def __new__(cls, subtype=None, closed: IntervalClosedType | None = None):
+    def __init__(self, subtype=None, closed: IntervalClosedType | None = None) -> None:
```

Comment:
should the PeriodDtype `__new__` be changed too?

Reply:
I think PeriodDtype has a `PeriodDtype.__cinit__` that is supposed to be "private", so I think it should continue to have it's own `__new__`